.circleci/config.yml

```diff
@@ -44,10 +41,8 @@ jobs:
         executor: default
         steps:
             - *attach_workspace
-            # - checkout # Commenting this line out as I don't think it is needed, please uncomment if the release fails
```

Comment:
@MrSwitch don't know if this isn't needed yet. waiting to see if https://github.com/5app/region-sync publishing works once the PR is approved.

Reply:
Ok, i'll put it back and update the comment based on your comments and if https://github.com/5app/region-sync/pull/1 works (then the next person to do a private repo can confirm either way). (read the bash but can't tell if it's needed.)